src/services/dashboards/modules/DashboardCloneModal.vue

```diff
@@ -133,16 +133,21 @@ export default defineComponent<Props>({
             viewers(value: DashboardViewer) { return value.length ? '' : i18n.t('DASHBOARDS.FORM.REQUIRED'); },
         });
 
-        const dashboardDetailInfoStore = useDashboardDetailInfoStore();
         const state = reactive({
             proxyVisible: props.visible,
             filteredVisibilityList: computed(() => visibilityList),
-            isProjectDashboard: computed(() => Object.prototype.hasOwnProperty.call(props.dashboard ?? {}, 'project_id')),
+            projectId: computed(() => {
+                if (Object.prototype.hasOwnProperty.call(props.dashboard ?? {}, 'projectId')) {
```

Comment:
Question: I'm curious about the difference between `props.dashboard.hasOwnProperty('projectId')`!